tests/parametric/test_dynamic_configuration.py

```diff
@@ -89,6 +89,22 @@ def set_and_wait_rc(test_agent, config_overrides: Dict[str, Any]) -> Dict:
     return test_agent.wait_for_rc_apply_state("APM_TRACING", state=2, clear=True)
 
 
+def set_and_wait_rc_telemetry(test_agent, config_overrides: Dict[str, Any]) -> Dict:
```

Comment:
[Question] What's the difference between this function and `set_and_wait_rc`. Could not we reuse the old one?

Reply:
this one returns payload from `app-client-configuration-change`, instead of rc_apply_state as in `set_and_wait_rc`. It also skips the first payload coming from golang since it's only supposed to contain some default configurations. But along the same vein, I see this is a difference in behaviour that we should look into to unify across the languages.